code/game/objects/items/melee/energy.dm

```diff
@@ -5,9 +5,12 @@
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 30)
 	resistance_flags = FIRE_PROOF
 	var/brightness_on = 3
+	total_mass = 0.4 //Survival flashlights typically weigh around 5 ounces.
+
 
 /obj/item/melee/transforming/energy/Initialize()
 	. = ..()
+	total_mass_on = w_class_on * 0.75

```

Comment:
?????
I'm 90% sure total_mass_on gets defined manually. This line of code right here will always override total_mass_on whether you like it or not
At the very least, give it a
```suggestion
	total_mass_on = (total_mass_on ? total_mass_on : (w_class_on * 0.75))
```